homeassistant/components/velbus/config_flow.py

```diff
@@ -46,24 +48,76 @@ async def async_step_user(
         self, user_input: dict[str, Any] | None = None
     ) -> ConfigFlowResult:
         """Step when user initializes a integration."""
-        self._errors = {}
+        return self.async_show_menu(
+            step_id="user", menu_options=["network", "usbselect"]
+        )
+
+    async def async_step_network(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle network step."""
+        if user_input is not None:
+            self._title = "Velbus Network"
```

Comment:
As multiple config entries are possible, can we set a title so the user knows which config entry it is?

Reply:
this is opposite to what was suggested in a previous PR

https://github.com/home-assistant/core/pull/131377

![image](https://github.com/user-attachments/assets/8622bc2f-c0d3-4176-b92b-3b31d995ff5b)